lib/bundler/gem_version_promoter.rb

```diff
@@ -1,9 +1,11 @@
 # frozen_string_literal: true
 module Bundler
+  # MODO: docs
```

Comment:
there's too many TODOs throughout the codebase, I'm using MODO (like 'chrisMO DO') to custom filter my editor down to ones that are 'mine'. it's goofy, i know.